homeassistant/components/rachio/device.py

```diff
@@ -19,11 +22,26 @@
     KEY_STATUS,
     KEY_USERNAME,
     KEY_ZONES,
+    MODEL_GENERATION_1,
+    SERVICE_PAUSE_WATERING,
+    SERVICE_RESUME_WATERING,
 )
 from .webhooks import LISTEN_EVENT_TYPES, WEBHOOK_CONST_ID
 
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_DEVICES = "devices"
+ATTR_DURATION = "duration"
+
+PAUSE_SERVICE_SCHEMA = vol.Schema(
+    {
+        vol.Optional(ATTR_DEVICES): cv.string,
+        vol.Optional(ATTR_DURATION, default=60): cv.positive_int,
```

Comment:
We normally try to allow users to configure longer time intervals using `cv.time_period` or `cv.positive_time_period`. Then convert the timedelta returned to whatever the api needs.

This integration seems to already use minutes for another service so the code owner should take the decision.

Reply:
@bdraco, what do you think?

Reply:
@brg468 I think keeping it only in minutes is fine in this case as the other service uses minutes, this is how the rachio presents the device, and making a mistake where you accidentally water for hours instead of minutes is less likely.